docs/playbooks/machine_reboot.md

```
@@ -35,11 +35,11 @@ the NICo site controller.
 **Example:**
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)